cdap-proto/src/main/java/io/cdap/cdap/proto/ScheduleDetail.java

```diff
@@ -36,7 +36,7 @@ public class ScheduleDetail {
 
   private final String namespace;
   private final String application;
-  private final String applicationVersion;
+  private final transient String applicationVersion;
```

Comment:
applicationVersion will be "-SNAPSHOT" always, so we want to hide this field in json response so it won't confuse users